### ISSUE TITLE
Picker's list has shadow in react native 0.39.

### ISSUE THREAD

**Issue by webdeveloper0429** [beefe/react-native-picker-android#14]
![untitled](https://cloud.githubusercontent.com/assets/22740206/20869900/f3e7a56c-ba8d-11e6-8561-bcde63f50c9c.png)
how do I fix this?

**Comment by shaikhussian** [comment]
@starcraft0429 Facing the issue in Picker List. If you have solved the issue please give the suggestions to solve the problem.

**Comment by webdeveloper0429** [comment]
@shaikhussian , do you have same issue?

**Comment by shaikhussian** [comment]
yes, I am also facing the same issue. Can you give me the suggestions ?

**Comment by webdeveloper0429** [comment]
I have no idea to solve it,
do you have any suggestion?

**Comment by svandenhout** [comment]
React-native from 0.36 and up has a problem where overflow: 'hidden' doesn't work without setting background: 'transparent' on the same component. I added background: 'transparent' to the styles of the correct components and the picker in node_modules/react-native-picker-android/index.js and everything looks fine now.

The project looks pretty abandoned so I won't try uploading a pull request.

**Comment by webdeveloper0429** [comment]
@svandenhout thanks for your reply.
so where style do I insert backgroundColor: 'transparent'?

**Comment by svandenhout** [comment]
@starcraft0429 At all the styles that apply overflow: 'hidden'

**Comment by webdeveloper0429** [comment]
Thank you a lot @svandenhout

**Comment by jrwm** [comment]
I have fixed it in my fork. Try it https://github.com/jrwm/react-native-picker-android

**Comment by webdeveloper0429** [comment]
@jrwm!
Did you fix issue on your github source code?
So is it possible to use your component without any adding style?

**Comment by jrwm** [comment]
@starcraft0429 yes, just replace 
`"react-native-picker-android": "^1.0.3"`
for 
`"react-native-picker-android": "git@github.com:jrwm/react-native-picker-android.git"`
in your package.json

**Comment by HotSpotNick** [comment]
@starcraft0429 you are the man
